### ISSUE TITLE
Research gateway box cloning solutions

### ISSUE THREAD

**Issue by arowla** [18F/openFEC#471]
cc @seanherron 

Need to figure out:
1. Whether we are going to clone from Oracle to Postgres or Oracle to Oracle
2. Whether we are going to use an off-the-shelf software package to do that, and
3. If so, what that package might be.

Such a software package would preferably be capable of doing both a full database wipe and update, as well as incremental nightly updates.

... And any other relevant research I'm forgetting. I know last week we planned to contact some support folks for answers on this. I don't think that has happened yet.

**Comment by rjmajma** [comment]
We found a possible solution in Golden Gate. Talking to FEC shortly about implementation and feasibility.

**Comment by rjmajma** [comment]
Waiting on the FEC team to determine feasibility of this solution. Will open up a new issue if we have to think of other options.
